app/lib/package/package_tags.dart

```diff
@@ -12,10 +12,20 @@ abstract class PackageTags {
 
   /// The first version of the package was published less than 30 days ago.
   static const String isRecent = 'is:recent';
+
+  static const $all = <String>[
+    isDiscontinued,
+    isNotAdvertized,
+    isRecent,
+  ];
 }
 
 /// Collection of version-related tags.
 abstract class PackageVersionTags {
   /// PackageVersion supports only legacy (Dart 1) SDK.
   static const String isLegacy = 'is:legacy';
+
+  static const $all = <String>[
+    isLegacy,
+  ];
```

Comment:
This is for parsing tags from the search query later.. hence I think it would be simpler if we expressed tags as a list of prefixes..

I think it would even be safe for us to express everything up-to the the colon... Hence, our tags prefix list might just be `['is:']`.

As we add more tags in `pana` we just put the prefixes into `pub-dev`, or expose them as a library from `pana`.